modules/gapi/misc/python/test/test_gapi_sample_pipelines.py

```diff
@@ -40,5 +93,182 @@ def test_mean_over_r(self):
                              'Failed on ' + pkg_name + ' backend')
 
 
+    def test_custom_mean(self):
+        img_path = self.find_file('cv/face/david2.jpg', [os.environ.get('OPENCV_TEST_DATA_PATH')])
+        in_mat = cv.imread(img_path)
+
+        # OpenCV
+        expected = cv.mean(in_mat)
+
+        # G-API
+        g_in = cv.GMat()
+        g_out = cv.gapi.mean(g_in)
+
+        comp = cv.GComputation(g_in, g_out)
+
+        pkg    = cv.gapi_wip_kernels((custom_mean, 'org.opencv.core.math.mean'))
```

Comment:
can it be `cv.gapi.wip_kernels` instead?

Reply:
...don't spend much time here if it is not

Reply:
No, it can't, all custom function placed in `cv2` package, I believe we work it out in future